_ingest-pipelines/processors/html-strip.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: default
+title: HTML strip
+parent: Ingest processors
+nav_order: 140
+---
+
+# HTML strip processor
```

Comment:
Global: Ensure H1 formatting consistency across these docs. gsub did not use code font, but script did, and we do not use it here.

Reply:
Revised across the latest ones to use regular font instead of code font. Published docs use reg font.